runtimes/azure-client-runtime/src/main/java/com/microsoft/azure/AzureEnvironment.java

```diff
@@ -83,9 +83,9 @@ public AzureEnvironment(
             "https://management.usgovcloudapi.net/");
 
     /**
-     * Provides the settings for authentication with Azure German Government.
+     * Provides the settings for authentication with Azure German.
      */
-    public static final AzureEnvironment AZURE_GERMAN_GOVERNMENT = new AzureEnvironment(
+    public static final AzureEnvironment AZURE_GERMAN = new AzureEnvironment(
```

Comment:
Where doe these names come from?  Just reading it, AZURE_GERMANY would make more sense.